standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -278,7 +279,7 @@
       "SELECT \"NWI_NEXT\" FROM \"NEXT_WRITE_ID\" WHERE \"NWI_DATABASE\" = ? AND \"NWI_TABLE\" = ?";
 
 
-  private List<TransactionalMetaStoreEventListener> transactionalListeners;
+  protected List<TransactionalMetaStoreEventListener> transactionalListeners;
```

Comment:
can you wrap method with if check, so you wouldn't need to change access modifier